src/image-sync-manager.ts

```diff
@@ -31,26 +32,31 @@ export class ImageSyncManager {
     mainPlayer: KalturaPlayerTypes.Player,
     imagePlayer: ImagePlayer,
     logger: KalturaPlayerTypes.Logger,
-    onSlideViewChanged: (viewChangeData: ExternalLayout) => void
+    onSlideViewChanged: (viewChangeData: ExternalLayout) => void,
+    onSlidesInitialized: () => void
   ) {
     this._eventManager = eventManager;
     this._mainPlayer = mainPlayer;
     this._imagePlayer = imagePlayer;
     this._logger = logger;
     this._onSlideViewChanged = onSlideViewChanged;
+    this._onSlidesInitialized = onSlidesInitialized;
     this._syncEvents();
     this._kalturaCuePointService = this._mainPlayer.getService('kalturaCuepoints');
   }
 
   private _syncEvents = () => {
     this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA_CHANGE, this._onTimedMetadataChange);
     this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
-    this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.FIRST_PLAYING, this._onFirstPlaying);
+    this._eventManager.listenOnce(this._mainPlayer, this._mainPlayer.Event.FIRST_PLAYING, this._onFirstPlaying);
   };
 
   private _onFirstPlaying = () => {
     this._firstPlaying = true;
     this._imagePlayer.preLoadImages();
+    if (this._imagePlayer.isImagesReady()) {
+      this._onSlidesInitialized();
+    }
```

Comment:
i am not sure that is the right way...
In live do we have images in the first play, not always.... only if already joined a media that had slides shown already.